tests/TestCase/Model/Behavior/SequenceBehaviorTest.php

```diff
@@ -9,6 +9,11 @@
 class Item extends Entity
 {
     protected $_accessible = ['*' => true, 'id' => false];
+    
+    protected function _getVirtualField()
```

Comment:
Even after making this change in test file the tests still pass without any changes to the behavior. So please make proper update to test file. The proper way to do that it add failing test case first and then update the code.